csharp/test/Ice/proxy/AllTests.cs

```diff
@@ -559,7 +560,7 @@ public static IMyClassPrx allTests(TestHelper helper)
             TestHelper.Assert(proxyProps["Test.ConnectionCached"].Equals("1"));
             TestHelper.Assert(proxyProps["Test.PreferNonSecure"].Equals("0"));
             TestHelper.Assert(proxyProps["Test.EndpointSelection"].Equals("Ordered"));
-            TestHelper.Assert(proxyProps["Test.LocatorCacheTimeout"].Equals("100"));
+            TestHelper.Assert(proxyProps["Test.LocatorCacheTimeout"].Equals("00:01:40"));
```

Comment:
Should these properties use the new format like `100s`, or am I missing something?

Reply:
This string is the result of `proxy.LocatorCacheTimeout.ToString()` where `LocatorCacheTimeout` is a `TimeSpan`.  While we implemented GetPropertyAsTimeSpan, we still need to decide and implement TimeSpan property value stringification. We'll have to fix this test once implemented.